scan.go

```diff
@@ -159,6 +161,8 @@ func newScanCommand() *cobra.Command {
 		"Show only packages that are used in code (requires code analysis)")
 	cmd.Flags().StringVarP(&csvReportPath, "report-csv", "", "",
 		"Generate CSV report of filtered packages")
+	cmd.Flags().BoolVarP(&reportDefectDojo, "report-defect-dojo", "", false, "Report to DefectDojo")
+	cmd.Flags().IntVarP(&defectDojoProductID, "defect-dojo-product-id", "", -1, "DefectDojo Product ID")
```

Comment:
Lets take `DEFECT_DOJO_HOST_URL` also as an arg. No point taking only that from environment. Also we should fail fast if it is not set instead of using a default `localhost` and end up failing later silently.